src/three/controls/PointerTracker.js

```diff
@@ -75,10 +75,22 @@ export class PointerTracker {
 
 	}
 
+	// get the pointer position in the coordinate system of the target element
+	getAdjustedPointer( e ) {
+
+		const rect = e.target.getBoundingClientRect();
+		const x = e.clientX - rect.left;
+		const y = e.clientY - rect.top;
+		return { x, y };
+
+	}
```

Comment:
I think we should provide a dom element to adjust the client x and y to be relative to as a member of the tracker. There's no real guarantee that the target is identical or the intended surface for each event.

We should also pass in a target object so we don't create a new object on each call.